src/client/entity-editor/validators/common.js

```diff
@@ -50,17 +66,8 @@ export function validateAlias(value: any): boolean {
 	);
 }
 
-export function validateAliases(values: any): boolean {
-	let every = (object, predicate) => _.every(object, predicate);
-	if (Iterable.isIterable(values)) {
-		every = (object, predicate) => object.every(predicate);
-	}
-	else if (!_.isObject(values)) {
-		return false;
-	}
-
-	return every(values, (value) => validateAlias(value));
-}
+export const validateAliases = _.partial(
+	validateMultiple, _, validateAlias);
```

Comment:
Might be best practice to use `_.partial.placeholder` in case this code gets copied for anything on the client side.

Reply:
Yeah, this is a good idea anyway for the clarity.